optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -62,8 +62,9 @@
 
```

Comment:
Could you delete `if type_checking.TYPE_CHECKING:` in line 30?

Reply:
Okay.